### ISSUE TITLE
How can I output generation scores(logits)?

### ISSUE THREAD

**Issue by Sishxo** [ymcui/Chinese-LLaMA-Alpaca-2#541]
### Check before submitting issues

- [X] Make sure to pull the latest code, as some issues and bugs have been fixed.
- [X] I have read the [Wiki](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/wiki) and [FAQ section](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/wiki/FAQ) AND searched for similar issues and did not find a similar problem or solution
- [X] Third-party plugin issues - e.g., [llama.cpp](https://github.com/ggerganov/llama.cpp), [LangChain](https://github.com/hwchase17/langchain), [text-generation-webui](https://github.com/oobabooga/text-generation-webui), we recommend checking the corresponding project for solutions

### Type of Issue

Model inference

### Base Model

Chinese-Alpaca-2 (7B/13B)

### Operating System

Linux

### Describe your issue in detail

This is my generation config
```
generation_config = GenerationConfig(
    temperature=0.1,
    top_k=20,
    top_p=0.9,
    do_sample=True,
    num_beams=1,
    repetition_penalty=1.0,
    max_new_tokens=400,
    output_scores=True,
)
```
However, It didn't output logits as expected

### Dependencies (must be provided for code-related issues)

None

### Execution logs or screenshots

None
